src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs

```diff
@@ -159,7 +159,7 @@ private void RevealObjectValues(string origin, JToken resource, int level)
                 case JValue value:
                     if (ResourceFilter.NonCompareValues.All(func => !func(value.ToString())))
                     {
-                        Assert.Contains(value.ToString(), origin);
+                        Assert.Contains(value.ToString().Trim(), origin);
```

Comment:
Just curious what does this change used for?

Reply:
Since we added the following property:

![image](https://user-images.githubusercontent.com/9417900/104990804-57b47c00-5a58-11eb-8b01-da2b4d10d6b7.png)

There will be a `"valueString": "  "` in the result JSON, which contains spaces, which should be trimmed before comparing.

Reply:
Got it. Thanks!